app/views/layouts/application.html.erb

```diff
@@ -4,30 +4,11 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title><%= content_for?(:title) ? yield(:title) : "TrainHard" %></title>
+    <title><%= content_for?(:title) ? yield(:title) : "StockTracker" %></title>
```

Comment:
we should avoid putting logic in the views, consider moving this into helper/decorator